src/OrbitGl/TriangleToggle.h

```diff
@@ -17,13 +17,10 @@ class Track;
 class TriangleToggle : public orbit_gl::CaptureViewElement,
                        public std::enable_shared_from_this<TriangleToggle> {
  public:
-  enum class State { kInactive, kExpanded, kCollapsed };
-  enum class InitialStateUpdate { kKeepInitialState, kReplaceInitialState };
-
-  using StateChangeHandler = std::function<void(TriangleToggle::State)>;
-  explicit TriangleToggle(State initial_state, StateChangeHandler handler, TimeGraph* time_graph,
-                          orbit_gl::Viewport* viewport, TimeGraphLayout* layout, Track* track,
-                          float size);
+  using StateChangeHandler = std::function<void(bool)>;
+  explicit TriangleToggle(bool is_collapsed, bool is_collapsible, StateChangeHandler handler,
```

Comment:
Other possibility could be have both at the end and is_collapsed set to false by default. I would like to not have this value for every track, since that almost every track are expanded by default.

Reply:
I removed it from the constructor and defaulted it with collapsible and not collapsed.